src/VoerroTagsInput.vue

```diff
@@ -590,8 +590,9 @@ export default {
                 const compareable = this.caseSensitiveTags
                     ? tag[this.textField]
                     : tag[this.textField].toLowerCase();
+                const keys = this.searchResults.map((res) => (res.key));
```

Comment:
You should've used `this.idField` instead of `key`. I'll fix it myself because it'll be faster.